modules/environment-roles/templates/shared_iam_terraform_policy.json.tpl

```diff
@@ -69,6 +70,7 @@
         "arn:aws:iam::${account_id}:role/TDRApiGatewayCloudwatchRole${title(environment)}",
         "arn:aws:iam::${account_id}:role/TDRApiUpdateRole",
         "arn:aws:iam::${account_id}:role/TDRBackendChecks*Role${title(environment)}",
+        "arn:aws:iam::${account_id}:role/TDRMetadataChecks${title(environment)}-role",
```

Comment:
Is there a reason to use `-role` instead of `Role` in the IAM role name?

For consistency the format of the role name should be: `TDR{role name}{environment}Role`

Reply:
The reason is that this is how role names are created in the `da-terraform-modules` step function module. I can switch to using `tdr-terraform-modules` if this is a big concern?

Reply:
Fair enough.